### PR TITLE
Hex color code embed 

### PR DESCRIPTION
**Pull request by Syycorax**
### This code is currently **UNTESTED**
Send an embed when a hex color code (#ffffff) is sent

### CODE REVIEW

File: cogs/omega.py [hunk 1]
```diff
@@ -98,6 +99,18 @@ async def on_message(self, message):
         if message.author.bot:
             return
 
+        # Checks if the message is an hex code
+        if re.match("^#([A-Fa-f0-9]{6})$", message.content):
+            #Removes the '#'
+            hex_code = message.content[1:]
+            x = requests.get('https://www.thecolorapi.com/id?hex='+ hex_code).json()
```
Comment (Tim-ats-d) [inline cogs/omega.py]: You should name variable `x` by a more explicit name (like `req` or `res`)

File: cogs/omega.py [hunk 2]
```diff
@@ -98,6 +99,18 @@ async def on_message(self, message):
...
+            title = x['name']['value']
+            img = x['image']['bare']
+            rgb = x['rgb']['value']
+            embed = Embed(title='Color #' + title + rgb, color='0x' + hex_code)
```
Comment (Tim-ats-d) [inline cogs/omega.py]: Use f-string
```suggestion
            embed = Embed(title=f"Color #{title}{rgb}", color=f"0x{hex_code}")
```

File: cogs/omega.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 from datetime import datetime
 import re
 from typing import AsyncGenerator
+import requests
```
Comment (Tim-ats-d) [inline cogs/omega.py:5]: Import need sorted by alphabetical order.

**Review by Tim-ats-d — CHANGES_REQUESTED** [review]

**Comment by Tim-ats-d** [comment]
You need to add a docstring at your method

**Review by Tim-ats-d — APPROVED** [review]

**Review by Guiguim255 — APPROVED** [review]
All good but why adding a blank line in the imports ?

**Pull request merged**